redbot/cogs/audio/converters.py

```diff
@@ -25,50 +27,46 @@
     "get_playlist_converter",
 ]
 
-if TYPE_CHECKING:
-    _bot: Red
-    _config: Config
-else:
-    _bot = None
-    _config = None
+T_ = _
+_ = lambda s: s
 
-_SCOPE_HELP = """
+_SCOPE_HELP: Final[str] = _(
+    """
 Scope must be a valid version of one of the following:
 ​ ​ ​ ​ Global
 ​ ​ ​ ​ Guild
 ​ ​ ​ ​ User
 """
-_USER_HELP = """
+)
+_USER_HELP: Final[str] = _(
+    """
 Author must be a valid version of one of the following:
 ​ ​ ​ ​ User ID
 ​ ​ ​ ​ User Mention
 ​ ​ ​ ​ User Name#123
 """
```

Comment:
I'm sorry Draper >.>
```suggestion
"""
)
```

Reply:
Don't worry I'm just pressing buttons